src/Microsoft.Health.Fhir.CosmosDb/Features/Storage/Registry/CosmosDbSearchParameterStatusDataStore.cs

```diff
@@ -39,11 +39,11 @@ public sealed class CosmosDbSearchParameterStatusDataStore : ISearchParameterSta
             _queryFactory = queryFactory;
         }
 
-        public async Task<IReadOnlyCollection<ResourceSearchParameterStatus>> GetSearchParameterStatuses()
+        public async Task<IReadOnlyCollection<ResourceSearchParameterStatus>> GetSearchParameterStatuses(CancellationToken cancellationToken)
```

Comment:
Super nit: do you think it would be useful to add a comment here to explain why we have two means of cancellation? Or maybe we could rename `cancellationSource` to add clarity?

Reply:
Yep, makes sense. I changed the name to retryDelayToken as that seems to be it's purpose.